pkg/cluster/k8sres.go

```diff
@@ -1052,6 +1052,11 @@ func (c *Cluster) generateCloneEnvironment(description *acidv1.CloneDescription)
 func (c *Cluster) generatePodDisruptionBudget() *policybeta1.PodDisruptionBudget {
 	minAvailable := intstr.FromInt(1)
 
+	// Avoid creating an unsatisfyable budget.
+	if c.Spec.NumberOfInstances <= 1 {
```

Comment:
I'd change it to < 1. A minAvailable: 1 for a cluster with one pod is easy to satisfy.